docs/commands.md

```diff
@@ -102,4 +102,79 @@ can be used to hint the user e.g. where to run `init` or `validate` from.
 }
 ```
 
+### `module.calls`
+
+List of module which are called from the current module.
```

Comment:
```suggestion
List of modules called by the module under the given URI.

Empty array may be returned when e.g.
  - the URI doesn't represent a module
  - the configuration is invalid
  - modules are not installed
  - there are no module calls

The data is sourced from the local cache within `.terraform` and so it may not necessarily represent newly added module calls in the configuration until they're installed via `get` or `init`.
```